qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -619,6 +619,17 @@
                 devcon_dirname += "amd64"
                 qemu_ga_pkg = qemu-ga-x64.msi
             devcon_path = "WIN_UTILS:\devcon\${devcon_dirname}\devcon.exe"
+        - gagent_debugview_VSS_DLL:
+            only Windows
+            no isa_serial
+            gagent_check_type = debugview_VSS_DLL
+            virtio_win_media_type = iso
+            cdroms += " virtio"
+            clone_master = yes
+            master_images_clone = image1
+            remove_image_image1 = yes
+            cmd_run_debugview = 'start WIN_UTILS:\Debugviewconsole.exe -d C:\debug.dbglog'
+            cmd_check_string_VSS = 'type C:\debug.dbglog | findstr /i "QEMU Guest Agent VSS Provider"'
```

Comment:
@6-dehan Will it be more accurate to append "\[\d+\]" to findstr /i "QEMU Guest Agent VSS Provider"'? I am not sure about the accurate pattern, just something match "QEMU Guest Agent VSS Provider**[10552]**", including the integers.

Reply:
I don't think we need to, even 'QEMU Guest Agent VSS Provider' can identify as the unique identifier that program Deviewconsole.exe run well.

Reply:
@6-dehan I suppose it could possibly raise any error or warning with this prefix, which indicate a failure, so more accurate would be better.

Reply:
okay, I got it. updating it.